.github/workflows/vulncheck.yaml

```diff
@@ -0,0 +1,36 @@
+---
+name: Go Vulnerability detection
+
+on:
+  push:
```

Comment:
probably we dont need `push`, just `pull_request` for now.

Reply:
Umm... The PRs will not be blocked but the job will keep failing, won't it? If so I think we should hold this until the failure could be fixed.
I would like to avoid the "Let's ignore the failure as it is a known failure" as much as possible because nobody takes care of it in the end.

Reply:
We two options:
a) Introduce this when all vulnerabilities are fixed.
b) Make it return 0 so it passes while also reporting the issues (like a warning) wdyth?

Reply:
Btw if we use 1.19.8 this does not report any issues and it passes, check my comment here: https://issues.redhat.com/browse/SRVCOM-2391.

Reply:
Is anything preventing us from using 1.19.8? If not, let's use that and merge this 👍

Reply:
It is a decision of the productization team to make. rpm builds still dont support 1.19.x in general.

Reply:
Hm that has to be resolved if we want to mitigate the CVEs. Anyways, IMHO let's hold this until we can bump to go 1.19.8, then we do have the benefits, without the unnecessary warnings.

Reply:
Or just do https://github.com/knative/actions/pull/136 so we avoid failures until the tool gets more flexible.